app/controllers/users_controller.rb

```diff
@@ -1,12 +1,10 @@
-class UsersController < ApplicationController
-  def new
-    @user = User.new
-  end
+class UsersController < Clearance::UsersController
 
   def create
-    @user = User.new(user_params)
+    @user = user_from_params
     if @user.save
-      session[:user_id] = @user.id
+      sign_in @user
+      flash[:notice] = "Sign Up successful! You rock."
       redirect_to dashboard_path
```

Comment:
Whoops, I had a typo in the original (updated now): I was missing the `notice:` key to `redirect_to`.